server/gifts/giftController.js

```diff
@@ -44,8 +44,13 @@ module.exports = {
       promises.push(amazonSync(ASIN));
     });
     Promise.all(promises).then(function(result){
+      console.dir(result);
```

Comment:
can remove console
?TODO: edit this to accommodate other gift types besides books